labs/src/main/scala/org/scalalabs/basic/lab03/RecursionPatternMatchingExercise.scala

```diff
@@ -69,7 +87,7 @@ object RecursionPatternMatchingExercise {
    * List(List(1,2,3), List('A, 'B, 'C), List('a, 'b, 'c)) -> List(List(1, 'A, 'a), List(2, 'B, 'b), List(3, 'C, 'c))
    */
   def zipMultiple(in: List[List[_]]): List[List[_]] = {
-    error("fix me")
+    in.transpose
```

Comment:
Thanks for the tip on this one. I couldn't figure out how to extend it to zipMultipleWithDifferentSize, though.